nano/core_test/active_transactions.cpp

```diff
@@ -523,8 +523,32 @@ TEST (active_transactions, update_difficulty)
 		ASSERT_NO_ERROR (system.poll ());
 	}
 	// Update work with higher difficulty
-	node1.work_generate_blocking (*send1, difficulty1 + 1);
-	node1.work_generate_blocking (*send2, difficulty2 + 1);
+	auto work1 = node1.work_generate_blocking (genesis.hash (), difficulty1 + 1, boost::none);
```

Comment:
probably `send1->root ()` is easier to undershand here? Same for below

Reply:
Done